OpenSim/Simulation/Model/ContactMesh.cpp

```diff
@@ -37,7 +37,7 @@ ContactMesh::ContactMesh() :
     setupProperties();
 }
 
-ContactMesh::ContactMesh(const std::string& filename, const SimTK::Vec3& location, const SimTK::Vec3& orientation, Body& body) :
+ContactMesh::ContactMesh(const std::string& filename, const SimTK::Vec3& location, const SimTK::Vec3& orientation, PhysicalFrame& body) :
```

Comment:
Then I would have to also rename the data member to `frame` and which would also cause a request to change the property to `frame_name` which would beg the question why isn't it using a Connector to a Frame instead? And also why ContactGeometry doesn't simply require Frame now that we have OffsetFrames we don't need pass along location and orientation all over the place. Given that we are likely to revamp contact geometry (post dev week) to leverage Geometry I thought i would leave it alone. I could address making it coniststently use frame (instead of body) as a quick side PR.